src/utils/logging-buffer.ts

```diff
@@ -83,35 +89,35 @@ export class LogBuffer {
     if (this.buffer.length === 0) {
       return;
     }
-
+  
     const logsToSend = [...this.buffer];
     this.buffer = [];
-
+  
     try {
-      const response = await fetch(`${this.acrolinxUrl}/int-service/api/v1/logs`, {
-        method: 'POST',
-        headers: {
-          'Content-Type': 'application/json',
-        },
-        body: JSON.stringify({
+      await post<Response>(
+        this.constructFullPath('/logs'),
+        {
           appName: 'sidebar-client-app',
```

Comment:
Not all requests will come from sidebar.
sdk-js also get used by other application like CLI & content analyzer. Also if all log will say sidebar-client-app then there is no differentiation between AEM or oxygen integration.